nservicebus/messaging/discard-old-messages.md

```diff
@@ -49,7 +49,9 @@ TimeToBeReceived relies on underlying functionality in the transport infrastruct
 
 ### MSMQ transport
 
-MSMQ continuously checks the TimeToBeReceived of all queued messages. As soon as the message has expired, it is removed from the queue, and disk space reclaimed. MSMQ will however only allow a single TimeToBeReceived for all messages in a transaction. If multiple messages enlist TimeToBeReceived from the first message will be used for all messages in the same transaction even if they differ, leading to a potential message loss scenario. To prevent this usage of TimeToBeReceived is not allowed for endpoints with [transaction mode](/nservicebus/transports/transactions.md) `TransportTransactionMode.AtomicSendsWithReceive` or higher.
+MSMQ continuously checks the TimeToBeReceived of all queued messages. As soon as the message has expired, it is removed from the queue, and disk space reclaimed. 
+
+MSMQ will however only allows a single TimeToBeReceived for all messages in a transaction. If multiple messages enlist in a single transaction, then TimeToBeReceived from the first message will be used for all messages, leading to a potential message loss scenario. To prevent the message loss, `TimeToBeReceived` setting is not supported for endpoints with [transaction mode](/nservicebus/transports/transactions.md) `TransportTransactionMode.AtomicSendsWithReceive` or `Transaction Scope (Distributed Transaction)`.
```

Comment:
allows => allow
To prevent the message loss => remove "the"

`TimeToBeReceived` setting => remove "setting"?